### PR TITLE
[Enhancement] ADD RowSerializer for doris flink connector

### PR DESCRIPTION
**Pull request by dinggege1024**
# Proposed changes

Issue Number: #70 

## Problem Summary:

ADD RowSerializer AND ut

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (Yes)
3. Has document been added or modified: (No,but will)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

### CODE REVIEW

**Comment by dinggege1024** [comment]
@JNSimba  Hi Simba, can you review this , thx~

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/RowSerializer.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        String valString;
+        if (JSON.equals(type)) {
+            valString = buildJsonString(rowDataRecord, maxIndex);
+        } else if (CSV.equals(type)) {
+            valString = buildCSVString(rowDataRecord, maxIndex);
+        } else {
+            throw new IllegalArgumentException("The type " + type + " is not supported!");
+        }
+        return valString.getBytes(StandardCharsets.UTF_8);
+    }
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/RowSerializer.java:78]: How about calling `RowDataSerializer.serialize(record)` directly?

Comment (dinggege1024) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/RowSerializer.java]: > How about calling `RowDataSerializer.serialize(record)` directly?

LGTM

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/RowSerializer.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    public String parseDeleteSign(RowKind rowKind) {
+        if (RowKind.INSERT.equals(rowKind) || RowKind.UPDATE_AFTER.equals(rowKind)) {
+            return "0";
+        } else if (RowKind.DELETE.equals(rowKind) || RowKind.UPDATE_BEFORE.equals(rowKind)) {
+            return "1";
+        } else {
+            throw new IllegalArgumentException("Unrecognized row kind:" + rowKind.toString());
+        }
+    }
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/RowSerializer.java:73]: This function seems to be useless?

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**